src/_angular/url-field/dtv-url-field.js

```diff
@@ -73,6 +93,10 @@
               scope.invalidType = "url";
             }
 
+            if (isValid) {
+              testImage();
```

Comment:
@donnapep `isValid` value is going to get returned regardless of `testImage()`. I see a potential conflict there. Should the value of `testImage()` be assigned to `isValid` instead? In `testImage`, instead of setting and applying `scope.valid`, it could return `true` or `false`.